src/ArchiveManagement/NexusMods.FileExtractor/Extractors/SevenZipExtractor.cs

```diff
@@ -137,6 +137,9 @@ private async Task ExtractAllAsync_Impl(IStreamFactory sFn, AbsolutePath destina
                     }, true)
                 .WithStandardOutputPipe(PipeTarget.ToDelegate(line =>
                 {
+                    if (string.IsNullOrWhiteSpace(line)) return;
+                    _logger.LogDebug("[7z.exe] {Line}", line);
```

Comment:
Looking at the highperformance logging stuff.
I set it to debug since the CI doesn't have trace enabled and the main purpose was to better debug errors from the CI.